src/servers/ZoneServer2016/entities/constructionchildentity.ts

```diff
@@ -469,6 +469,13 @@ export class ConstructionChildEntity extends BaseLightweightCharacter {
         ...parentFoundation.freeplaceEntities,
         ...this.freeplaceEntities,
```

Comment:
should be removed since you do it correctly below

Reply:
done